### ISSUE TITLE
COS loader won't open COS file

### ISSUE THREAD

**Issue by justincely** [spacetelescope/specviz#229]
When trying on: lc1s02vtq_x1d.fits

```python
ERROR:root:Incompatible loader for selected data: COS Fits
ERROR:root:Could not open file.
```

**Comment by pllim** [comment]
I think this was a common problem in one of the previous sprints. Data from the same instrument had to use different YAML files because they were generated from different software.

**Comment by justincely** [comment]
closed with #235
